components/homme/src/share/dimensions_mod.F90

```diff
@@ -3,47 +3,45 @@
 #endif
 
 module dimensions_mod
-#ifdef CAM
-  use constituents, only : qsize_d=>pcnst ! _EXTERNAL
+#if defined(CAM) && !defined(FVM_TRACERS)
+  use constituents, only : qsize_d => pcnst ! _EXTERNAL
 #endif
   implicit none
   private
 
 ! set MAX number of tracers.  actual number of tracers is a run time argument  
-#ifdef CAM
-#else
+#ifndef CAM
 #ifdef QSIZE_D
   integer, parameter         :: qsize_d=QSIZE_D
 #else
   integer, parameter         :: qsize_d=4
 #endif
 #endif
 
-
+#if defined(CAM) && defined(FVM_TRACERS) 
```

Comment:
Can you put this above, right next to the other mod? Keeping two versions of the same declaration next to each other helps visually parsing the code. E.g:

```cpp
#ifndef CAM
#ifdef FVM_TRACERS
  integer, parameter         :: qsize_d =10 ! CAM-SE tracers (currently CAM-SE supports 10 condensate loading tracers)
#else
 use constituents, only : qsize_d => pcnst ! _EXTERNAL
#endif
```

Reply:
Sure. Just committed the change.